docs/padroesDeProjeto/correcoes_E2.md

```diff
@@ -0,0 +1,11 @@
+# Correções
```

Comment:
Esse documento já foi criado e está sendo editado na branch de design patterns

Reply:
Já editei ele na outra branch, quando for mergear ela é so sobrescrever.